common.js

```diff
@@ -0,0 +1,46 @@
+"use strict";
+
+const js = require("@eslint/js");
+const eslintConfigPrettier = require("eslint-config-prettier");
+
+const recommended = js.configs.recommended;
+
+module.exports = {
```

Comment:
I'd love to revisit these at some point, I suspect some of them are no longer needed or turned on by default anyway. But definitely out of scope for this PR.